contracts/contracts/core/BidderRegistry.sol

```diff
@@ -114,8 +116,11 @@ contract BidderRegistry is
             maxBidPerBlock[msg.sender][window] =
                 newLockedFunds /
                 WindowFromBlockNumber.BLOCKS_PER_WINDOW;
-
+            
             emit BidderRegistered(msg.sender, newLockedFunds, window);
+            emit BidderDepositedForWindow(msg.sender, window, amountToDeposit, newLockedFunds);
+            emit MaxBidForWindowUpdated(msg.sender, window, maxBidPerBlock[msg.sender][window]);
```

Comment:
I am not sure why this separate event is needed. The max bid per block is function of window length and amount. So it makes sense to include it in the same event above.

Reply:
Yeah since it's specific to the bidder makes sense to just consolidate